.github/workflows/release.yml

```diff
@@ -0,0 +1,43 @@
+# Cut a release whenever a new tag is pushed to the repo.
+# You should use an annotated tag, like `git tag -a 1.2.3`
+# and put the release notes into the commit message for the tag.
```

Comment:
This is a tag for this new repository.  When would we push a tag to it?  When we make a jq release?

Reply:
This repo versioning doesn't need to match JQ releases; for this repo, it only needs to be monotonically increasing.

I'd at least tag this one `0.0.1` when this PR3 merges.